Example/ExampleAPI/Endpoint/Base/JsonEndpoint.swift

```diff
@@ -14,13 +14,23 @@ protocol JsonEndpoint: Endpoint, URLRequestBuildable where Content: Decodable {}
 
 extension JsonEndpoint {
 
+    public typealias Failure = Error
```

Comment:
Xcode shows warning on this file.

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/410293/105138149-8e949b80-5b05-11eb-94b3-0a345e7034b2.png">